lib/sepa.test.js

```diff
@@ -123,6 +124,44 @@ describe('xml generation for transfer documents', () => {
     expect(debtorId).not.toBeUndefined();
     expect(debtorId.textContent).toBe(validDutchCreditorId);
   });
+  
+  test('ctry and address field not null', () => {
+    // GIVEN
+    const doc = validTransferDocument({debtorId: 'FR72ZZZ123456', debtorName: 'debtor-name'});
```

Comment:
Can you remove `debtorId` and `debtorName` here? The values are not relevant to the behavior.
I.e. write `validTransferDocument({})`